util/devel/test/apptainer/image-rebuild.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Rebuilds the apptainer image in each current subdirectory
+
+for name in current/*
+do
+  if [ -f $name/image.def ]
+  then
+    echo "cd $name && apptainer build --force --fakeroot image.sif image.def"
+    ( cd $name && apptainer build --force --fakeroot image.sif image.def )
```

Comment:
Don't we need to check if the build succeeds?

Reply:
If it fails, we will see an error in the output & subsequent operations on the image will fail. With these scripts, it's usually better not halt (and not build the subsequent images).